routemaster/models/event_data.rb

```diff
@@ -0,0 +1,36 @@
+require 'routemaster/models'
+require 'routemaster/mixins/assert'
+require 'msgpack'
+
+module Routemaster
+  module Models
+    # Value object for event data blobs.
+    # Fails on construction if the serialized representation is too long.
+    class EventData < SimpleDelegator
+      include Mixins::Assert
+
+      MAX_EVENT_DATA = ENV.fetch('ROUTEMASTER_MAX_EVENT_DATA').to_i
```

Comment:
There is one, in `.env`. Given this is an app, not a library, `.env` is pretty much guaranteed to be loaded and I didn't want to duplicate defaults?